src/main/kotlin/org/rust/ide/inspections/fixes/RemoveRefFix.kt

```diff
@@ -0,0 +1,30 @@
+package org.rust.ide.inspections.fixes
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiDocumentManager
+import org.rust.lang.core.psi.RustExprElement
+
+
+/**
+ * Fix that converts the given reference to owned value.
+ * @param argEl An element, that represents a reference from which the first
+ * symbol '&' must be removed.
+ * @param fixName A name to use for the fix instead of the default one to better fit the inspection.
+ */
+class RemoveRefFix(
+    val argEl: RustExprElement,
+    val fixName: String = "Change reference to owned value") : LocalQuickFix {
```

Comment:
We usually move `) :` on the new line. That way, the class body is more visible, and the extends list has a predictable position:

```
class RemoveRefFix(
    val argEl: RustExprElement,
    val fixName: String = "Change reference to owned value"
) : LocalQuickFix {
```